client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -52,6 +52,15 @@ vec4 applyToneMapping(vec4 color)
 }
 #endif
 
+#ifdef ENABLE_FAR_CONTRAST
+vec4 applyFarContrast(vec4 color)
+{
+	float cadist = vPosition.z/5000;
+	color.rgb = color.rgb*(cadist*9+1)-cadist*1.8;
```

Comment:
vec3 = vec3 * float - float;  does it even compile?
also again, integers instead of floats
spaces